pegasus/sites.v3/code.org/views/admins_email_signup_form.html

```diff
@@ -43,6 +41,6 @@
   </select>
 
   <input type=hidden id="lead_source" name="lead_source" value="Web Form - Administrators Page">
-  <div class="g-recaptcha" data-sitekey="6LezdGMkAAAAADpS_8wn3yJV8of06uQcYiXHanEp"></div>
-  <input type="submit" name="submit" value="Submit">
+  <div class="g-recaptcha" data-sitekey="6LezdGMkAAAAADpS_8wn3yJV8of06uQcYiXHanEp" data-callback="recaptchaCallback"></div>
```

Comment:
After talking with Dave, we might want to figure out a way to make this sitekey harder to access. I think we're safe for now because we're not allowing `studio.code.org` access to the recaptcha (for now), but wanted to flag it here –– I'll write up a doc tomorrow so that this is tracked.

Reply:
Could you link https://developers.google.com/recaptcha/docs/display as well for reference? It's looking like the `class="g-recaptcha` tag is doing some important stuff and I don't want the details to get lost

Reply:
Great job using the `data-callback` functionality for validation –– this is a very clean solution imo!

Reply:
Awesome, thanks for writing this up @megcrenshaw!